src/layouts/Layout.jsx

```diff
@@ -35,6 +63,13 @@ class Layout extends React.PureComponent {
       <div>
         <Grid fluid={ true }>
           <Row>
+            <Navbar style={style.Navbar}>
```

Comment:
@invinciblycool are you testing this locally? 

This is not a valid React Component.